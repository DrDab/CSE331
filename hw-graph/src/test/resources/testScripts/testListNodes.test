# Create graphs w/ 0, 1, 2 nodes.
CreateGraph g0

CreateGraph g1
AddNode g1 ALPHA

CreateGraph g2
AddNode g2 ALPHA
AddNode g2 BRAVO

# Test listing nodes on graphs w/ 0, 1, 2 nodes.
ListNodes g0
ListNodes g1
ListNodes g2