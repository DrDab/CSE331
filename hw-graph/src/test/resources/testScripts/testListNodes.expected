# Create graphs w/ 0, 1, 2 nodes.
created graph g0

created graph g1
added node ALPHA to g1

created graph g2
added node ALPHA to g2
added node BRAVO to g2

# Test listing nodes on graphs w/ 0, 1, 2 nodes.
graphName contains:
graphName contains: ALPHA
graphName contains: ALPHA BRAVO